src/app/main/component/ubs/components/ubs-order-details/ubs-order-details.component.scss

```diff
@@ -441,3 +442,28 @@ li {
     font-size: 0.65rem;
   }
 }
+
+@media (max-width: 992px) {
```

Comment:
It's better to use standard measures for media queries
https://gist.github.com/gokulkrishh/242e68d1ee94ad05f488